src/engine/specification.md

```diff
@@ -9,7 +9,12 @@ This specification is based on [Ethereum JSON-RPC API](https://eth.wiki/json-rpc
 * Message format and encoding notation
 * [Error codes improvement proposal](https://eth.wiki/json-rpc/json-rpc-error-codes-improvement-proposal)
 
-Client software **MUST** expose Engine API at a port independent from JSON-RPC API. The default port for the Engine API is 8550 for HTTP and 8551 for WebSocket.
+Client software **MUST** expose Engine API at a port independent from JSON-RPC API.
+The default port for the Engine API is 8550 for HTTP and 8551 for WebSocket.
+The Engine API is exposed under the `engine` namespace.
+
+To facilitate an Engine API consumer to access state and logs (e.g. proof-of-stake deposits) through the same connection,
+the client **MAY** also expose the `eth` namespace. 
```

Comment:
```suggestion
the client **MUST** also expose the `eth` namespace. 
```

Suggesting a MUST because otherwise interoperability of EL client software can't be guaranteed and CL will have to be able to handle different EL client behaviour wrt `eth` namespace exposure. Thus, I think using MAY hinders the benefit of the proposal itself.